docs/world-competition.md

```diff
@@ -0,0 +1,74 @@
+
+SoccerSim challenge this year includes:
+* Simulation of matches
+* Presentation of team description papers
+* Solving Technical Challenges.
+
+As usual, teams will be asked to submit the code of their robot players.
+We have frozen the rcj-soccersim software which is [available as a tagged release
+at our GitHub](https://github.com/RoboCupJuniorTC/rcj-soccersim/releases/tag/v2.1). This release will be used for simulating the matches.
+
+In order to spread the knowledge, we ask teams to provide team
+description papers. These might contain descriptions of team members’ roles,
+various approaches you have tried while implementing the code for
+simulated robots as well as anything you consider worth mentioning.
+For inspiration, feel free to
+[check last year's papers out](https://github.com/RoboCupJuniorTC/awesome-rcj-soccer#soccersim)!
+
+There will be at most two technical challenges.
+rcj-soccersim software will be updated appropriately for these challenges.
+There will be several days reserved for you to implement your solution
+and submit recorded video. Committee will then review it and evaluate each team.
+
+## Rules
+
+The rules are based on the regular RoboCupJunior Soccer rules, adapted to the
+simulation environment.
+
+You can find them at
+https://github.com/RoboCupJuniorTC/soccer-rules-simulation/blob/master/rules.pdf
+
+**Note**: The rules haven't changed much from the last year. Perhaps the most
+important changes include the change of the world's coordinate system and
+removal of exact GPS coordinates of the ball. Robots are forced to sense the ball
+using IR sesnsors. We encourage you to check out the sample Robot controllers
+as well as the [Webots](https://cyberbotics.com/doc/reference/worldinfo) documentation.
+
+## Documentation
+
+We now have two fully fledged documentation articles that can help you get
+started programming your robots:
+
+- [How to program your robot](https://robocupjuniortc.github.io/rcj-soccersim/how_to_robot/)
+- [Inter-robot communication](https://robocupjuniortc.github.io/rcj-soccersim/communication_between_robots/)
+
+If you feel like there is something missing in the documentation, do not
+hesitate to suggest a change -- Pull Requests are very much welcome!
+
+## Schedule
+
+_Unless otherwise specified, all dates use the [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth) timezone_
+
+* **June 1st:** Open Tournament Submission Form
+* **June 27th:** Tournament Submission Deadline
+* **June 27th:** Technical Challenge 1 Handout
+* **July 4th:** Challenge 1 Deadline
+* **July 12th:** Publishing Solutions of Challenge 1 as Videos on the Forum
+* **July 12th:** Team Description Paper Deadline
+* **July 12th:** Challenge 2 Handout
+* **July 13th .. 16th:** Publishing Tournament Results
+* **July 15th:** Challenge 2 Deadline
+* **July 23th:** Publishing Solutions of Challenge 2 as Videos on the Forum
+
+## Registration
+
+Registration form will be published soon!
```

Comment:
@Adman I guess at this point we should simply add it in here.

Reply:
Added